meta-intel-iot-security/meta-security-smack/recipes-core/systemd/systemd/0001-core-smack-add-support-for-defining-smack-rules-path-v229.patch

```diff
@@ -0,0 +1,145 @@
+From f233cd7f9a1bc27bafdcc89ef77987c15131c4b7 Mon Sep 17 00:00:00 2001
+From: Jaska Uimonen <jaska.uimonen@intel.com>
+Date: Wed, 16 Mar 2016 15:12:09 +0200
+Subject: [PATCH] core/smack: add support for defining smack rules path in
+ configure
+
```

Comment:
The description of the change should be in the patch header, in case we want to submit it upstream.

Also needs an Upstream-Status. What that status is depends on whether there are plans to submit it upstream.